OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -38,13 +38,16 @@ public class WarheadArgs
 		public WeaponInfo Weapon;
 		public int[] DamageModifiers = { };
 		public WPos? Source;
+		public WRot ImpactRotation;
```

Comment:
Would `ImpactOrientation` still make sense here? This would make it consistent with the actor orientation naming.